docs/esa/iso/iso.rst

```diff
@@ -323,34 +326,26 @@ Either by invocation of the URL provided in every row of previous section or usi
 
 Images can be displayed by using the following code:
 
-.. code-block:: python
+.. Skipping plotting example
+.. doctest-skip::
```

Comment:
it's OK to skip it for now, but for the a follow-up, this may be turned to a plot directive example, so we still run it.

Reply:
What is a plot directive example?

Reply:
It's the mpl sphinx extension `.. plot::`, photutils and astropy.modeling has a few good examples.